src/velero/values/upstream-values.yaml

```
@@ -14,3 +14,9 @@ initContainers:
     volumeMounts:
       - mountPath: /target
         name: plugins
+  - name: velero-plugin-for-azure
+    image: velero/velero-plugin-for-microsoft-azure:v1.1.0
```

```suggestion
    image: velero/velero-plugin-for-microsoft-azure:v1.10.0
```

Looks like this is failing CI currently - I think just because of the image ref not matching what is in the zarf.yaml.